### ISSUE TITLE
i thought i supposed to clear VRAM from comfyui

### ISSUE THREAD

**Issue by Amit30swgoh** [ntdviet/comfyui-ext#4]
![image](https://github.com/ntdviet/comfyui-ext/assets/42430868/ebe7e48f-e1ce-4175-979c-80cc3f347c56)

**Comment by S0LTIS** [comment]
Yep, and it does. Every time you queue it up the checkpoint needs to be reloaded. This is not healthy for the hardware doing the loading and unloading. It will wear down the hardware and drives. I immediately removed it bc it does exactly as it says.
